h/static/scripts/group-forms/components/CreateGroupForm.tsx

```diff
@@ -126,6 +126,7 @@ export default function CreateGroupForm() {
 
   const createGroup = async (e: SubmitEvent) => {
     e.preventDefault();
+    setErrorMessage('');
```

Comment:
I'm not sure it's worth the bother of writing a test for this?

Reply:
I don't think a test is especially necessary here. The code is at least executed by existing tests.